### PR TITLE
After supporting WordVecSpaceDisk

### PR DESCRIPTION
**Pull request by prasannababuAddagiri**

### CODE REVIEW

**Comment by prashanthellina** [comment]
@chatrapathik  @shivam-deepcompute Can you please review and tell me?

File: wordvecspace/command.py [hunk 1]
```diff
@@ -94,17 +107,17 @@ def define_subcommands(self, subcommands):
         convert_cmd.set_defaults(func=self.convert)
         convert_cmd.add_argument('input_dir',
             help='Input directory containing Google Word2Vec format files'
-                 ' (vocab.txt, vectors.bin)')
-        convert_cmd.add_argument('output_file',
-            help='Output file where WordVecSpace format files are produced')
+                                 ' (vocab.txt, vectors.bin)')
```
Comment (chatrapathik) [inline wordvecspace/command.py]: @prasannababuAddagiri Can you please check the indentation here.

File: wordvecspace/server.py [hunk 2]
```diff
@@ -160,27 +165,28 @@ def get_distances(self, row_words_or_indices: Union[str, int, tuple, list], col_
...
+        get_distances(["andhra"]) => [[ 1.5418  0.7153  0.277  ...,  1.1657  1.0774  0.7036]]
+        get_distances(["andhra"], metric='euclidean') => [[ 1.756   1.1961  0.7443 ...,  1.5269  1.4679  1.1862]]
         '''
-        if self._type == 'mem':
+        if self._type == 'mem' or 'disk':
             return self.wv.get_distances(row_words_or_indices, col_words_or_indices=col_words_or_indices, metric=metric, raise_exc=raise_exc).tolist()
 
         return self.wv.get_distances(row_words_or_indices, col_words_or_indices=col_words_or_indices, raise_exc=raise_exc).tolist()
 
     def get_nearest(self, words_or_indices: Union[str, int, list, tuple], k: int=512, metric: Union[str, None]=None, combination: bool=False, raise_exc: bool=False) -> list:
```
Comment (chatrapathik) [inline wordvecspace/server.py]: @prasannababuAddagiri here we are not following the pep-8 standard's can you please check it once.

File: wordvecspace/disk.py [hunk 3]
```diff
@@ -182,3 +191,115 @@ def get_word_occurrences(self, words_or_indices, raise_exc=False):
...
+
+        ner = []
+        for dist in distances:
+            dist = pd.Series(dist.reshape((len(dist,))))
+            dist = dist.nsmallest(k).keys()
+            ner.append(list(dist))
+
+        if combination:
+            return list(set(ner[0]).intersection(*ner))
+
```
Comment (chatrapathik) [inline wordvecspace/disk.py]: @prasannababuAddagiri here the empty line is not necessary.

File: test.py [hunk 4]
```diff
@@ -8,6 +8,7 @@
 from wordvecspace import annoy
 from wordvecspace import disk
 
+
```
Comment (shivam-deepcompute) [inline test.py]: I think extra line space is not required, though auto pep8 by default includes 2 line space, it looks odd to me. Please check

File: wordvecspace/disk.py [hunk 5]
```diff
@@ -182,3 +191,115 @@ def get_word_occurrences(self, words_or_indices, raise_exc=False):
...
+        else:
+            return ner if isinstance(words_or_indices, (list, tuple)) else ner[0]
```
Comment (chatrapathik) [inline wordvecspace/disk.py]: @prasannababuAddagiri here we no need to check the else condition. If the condition satisfies we can return there itself.
Ex:
```python
>>> def test(x):
...     if x == 1:
...             return x
...     return x
```

File: wordvecspace/command.py (hunk 1 above)
Comment (shivam-deepcompute) [reply wordvecspace/command.py]: agree @chatrapathik

File: wordvecspace/command.py [hunk 6]
```diff
@@ -133,8 +146,10 @@ def define_subcommands(self, subcommands):
                         Eg: --eargs n_trees=1:index_fpath=/tmp\
                         (This is considered only when the type is annoy)')
 
+
 def main():
     WordVecSpaceCommand().start()
 
+
```
Comment (shivam-deepcompute) [inline wordvecspace/command.py]: please for line space @prasannababuAddagiri

File: wordvecspace/convert.py [hunk 7]
```diff
@@ -26,51 +27,59 @@ def iter_vectors(self):
...
 
-    def write(self, token, vec, occur):
-        self.out.add(token, vec, occur)
+    def write(self, token, occur, vec):
+        self.out.add(token, occur, vec)
 
     def close(self):
         self.out.close()
 
+
```
Comment (shivam-deepcompute) [inline wordvecspace/convert.py]: line space here too @prasannababuAddagiri

File: wordvecspace/server.py [hunk 8]
```diff
@@ -212,10 +219,10 @@ def __init__(self, _type, input_file, port, n_trees=N_TREES, metric=METRIC, inde
     def start(self):
         self.api = API(log=self.log)
         self.api.register(APIFunctions(self._type,
```
Comment (chatrapathik) [inline wordvecspace/server.py]: @prasannababuAddagiri here we can pass all arguments in single line no need to split them.

**Comment by prasannababuAddagiri** [comment]
@chatrapathik @shivam-deepcompute Thank you for your review. will do modification.

**Pull request merged**